Resources/Prototypes/Datasets/Names/vox.yml

```diff
@@ -13,7 +13,6 @@
   - Bahepra
   - Bahicrecrek
   - Bahikravre
-  - Bakabepa
```

Comment:
can you remove the other one on top of the list so they stay in alphabetical order?